lib/order.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'menu'
+require_relative 'text'
+class Order
+  attr_reader :current_order, :order_total
+
+  def initialize
+    @current_order = []
+  end
+
+  def select_item(menu, item, quantity)
+    item_included?(menu, item)
+  end
+
+  def calculate_price
+
+  end
+
+  def complete_order
+    # totalprice
+    Text.new.send_text
+    @current_order
+  end
+
+  private
+
+  def item_included?(menu, item)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `menu`. If it's necessary, use `_` or `_menu` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)